README.md

```diff
@@ -60,3 +60,30 @@ npx hardhat deploy --network <network>
 To deploy the Fractal contracts open a terminal and run:
 ```shell
 npx hardhat node
+```
+## NPM Package
+The core contracts in this repository are published in an NPM package for easy use within other repositories.
+
+To install the npm package, run:
+
+```shell
+npm i @fractal-framework/core-contracts
+```
+
+Including un-compiled contracts within typechain-types. Follow theses steps hardhat plug-in https://www.npmjs.com/package/hardhat-dependency-compiler
```

Comment:
typo on `theses` and would be good to hyperlink the url like `(these steps)[url]`